bin/getIntegrationAccessToken.js

```diff
@@ -109,9 +109,14 @@ module.exports = async (
 
       return response.access_token;
     } catch (e) {
-      if (e.error !== 'invalid_scope' || i === METASCOPES.length - 1) {
+      const { message: errorMessage = '' } = e;
```

Comment:
Would it maybe be better to have a default message instead of an empty string? Maybe `An unknown error has occurred`.?